### PR TITLE
feat: require minimum Qt6.7

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@ set(APPLE_SUPPRESS_X11_WARNING ON)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(QT 6.5 NAMES Qt6 COMPONENTS Core REQUIRED)
+find_package(QT 6.7 NAMES Qt6 COMPONENTS Core REQUIRED)
```
Comment (saw-jan) [inline CMakeLists.txt:21]: if we do this in PR https://github.com/owncloud/client/pull/11994. Then we might get the CI running for this PR
```suggestion
find_package(QT 6.5 NAMES Qt6 COMPONENTS Core REQUIRED)
```

**Comment by DeepDiver1975** [comment]
WTF?

image

File: .drone.star [hunk 2]
```diff
@@ -22,7 +22,7 @@ OC_UBUNTU = "owncloud/ubuntu:20.04"
 # Eventually, we have to use image built on ubuntu
 # Todo: update or remove the following images
 # https://github.com/owncloud/client/issues/10070
-OC_CI_CLIENT_FEDORA = "owncloudci/client:fedora-39-amd64"
+OC_CI_CLIENT_FEDORA = "owncloudci/client:fedora-41-amd64"
 OC_CI_SQUISH = "owncloudci/squish:fedora-39-7.2.1-qt66x-linux64"
```
Comment (saw-jan) [inline .drone.star:26]: you should also update the squish image to 8.0 as well

Comment (saw-jan) [reply .drone.star]: updated

**Comment by saw-jan** [comment]
client build failed because of some deprecation warnings:
https://drone.owncloud.com/owncloud/client/19992/4/3
```c
[239/452] Building CXX object src/gui/CMakeFiles/owncloudGui.dir/activitywidget.cpp.o
/drone/src/src/gui/activitywidget.cpp:90:34: warning: 'qAsConst<QMap<QString, OCC::NotificationWidget *>>' is deprecated: Use std::as_const() instead. [-Wdeprecated-declarations]
   90 |         for (const auto widget : qAsConst(_widgetForNotifId)) {
```

**Comment by saw-jan** [comment]
might be the unmatched system libraries in `owncloud-ci/client`

**Comment by DeepDiver1975** [comment]
I am actually considering to move this step over to GitHub.....

**Comment by saw-jan** [comment]
> I am actually considering to move this step over to GitHub.....

yeeah, maybe we could rely on ctest in github actions

**Comment by DeepDiver1975** [comment]
@saw-jan you want to work on that?

**Comment by saw-jan** [comment]
> @saw-jan you want to work on that?

yep, I can. I will remove the drone pipeline for ctest in this PR. That means we no longer use `owncloudci/client` image

**Comment by saw-jan** [comment]
> ctest is already being executed in github workflow -
> 
> https://github.com/owncloud/client/blob/9bc3844635a211e4883fadcf13e39830ea34d83a/.github/workflows/main.yml#L164

yeah, then I will remove ctest pipeline in drone.
Thanks :+1:

**Review by saw-jan — APPROVED** [review]

**Pull request merged**